addons/hr_presence/views/hr_employee_views.xml

```diff
@@ -32,4 +63,32 @@
             </xpath>
         </field>
     </record>
+
+    <record id="hr_employee_view_kanban" model="ir.ui.view">
```

Comment:
Same ID as the previous record (line 46), only one will be loaded.

Reply:
Thanks, I'll fix that